integrations/coinbase/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/CoinbaseActivityViewModel.kt

```diff
@@ -40,11 +40,13 @@ class CoinbaseActivityViewModel @Inject constructor(
     private val coinBaseRepository: CoinBaseRepositoryInt
 ) : ViewModel() {
 
-    private val _paymentMethodsUiState = MutableStateFlow<PaymentMethodsUiState>(PaymentMethodsUiState.LoadingState(true))
-    val paymentMethodsUiState: StateFlow<PaymentMethodsUiState> = _paymentMethodsUiState
+    private val _paymentMethodsUiState = MutableLiveData<PaymentMethodsUiState>(PaymentMethodsUiState.LoadingState(true))
+    val paymentMethodsUiState: LiveData<PaymentMethodsUiState>
+        get() = _paymentMethodsUiState
 
-    private val _baseIdForFaitModelCoinBase= MutableStateFlow<BaseIdForFaitDataUIState>(BaseIdForFaitDataUIState.LoadingState(true))
-    val baseIdForFaitModelCoinBase:StateFlow<BaseIdForFaitDataUIState> = _baseIdForFaitModelCoinBase
+    private val _baseIdForFaitModelCoinBase = MutableLiveData<BaseIdForFaitDataUIState>(BaseIdForFaitDataUIState.LoadingState(true))
+    val baseIdForFaitModelCoinBase: LiveData<BaseIdForFaitDataUIState>
```

Comment:
We should not mix approaches (`LiveData` vs `StateFlow`) for communicating between ViewModels and fragments in a single module. It's fine to use `StateFlow` everywhere in a feature module if one prefers it, but it should be consistent.

Personally, I found that flows have some problems with respecting view lifecycles, so I stick to LiveData for now.